src/compiler/crystal/macros/methods.cr

```diff
@@ -1077,6 +1082,22 @@ module Crystal
 
           value
         end
+      when "has_key?"
+        interpret_check_args do |key|
+          case key
+          when SymbolLiteral
+            key = key.value
+          when MacroId
+            key = key.value
+          when StringLiteral
+            key = key.value
```

Comment:
**polish:** I think these branches could be merged into one for a bit more concise expression:
```suggestion
          when SymbolLiteral, MacroId, StringLiteral
            key = key.value
```

Reply:
👍🏻 

i also changed it in the places where i copied it from: 1f30fc6d91c944669bc79f158b236b5384f4c3be